src/help/zaphelp/contents/start/concepts/sessionManagement.html

```diff
@@ -26,6 +26,13 @@
 		the <a href="httpsessions.html">Http Sessions</a> Extension.
 	</p>
 
+	<h3>
+		<a name="hasm">Http Authentication Session Management</a>
```

Comment:
Indeed, updated the pull request to use HTTP (will change the other entries as well, in another pull request).